en/pe-docs/docs/devops/automation-pipelines/inbuilt-templates.md

```
@@ -0,0 +1,80 @@
+
+??? "choreo/test-runner-build@v1"
+    Runs tests using the built-in test runner.
+
+??? "choreo/prism-build@v1"
+    Builds Prism projects.
+
+## Publish provisioned resource details to Choreo Marketplace
+
+At the end of all the resource provisioning steps in the automation pipeline, to publish metadata and credentials of provisioned resources, use inbuilt pipeline template `choreo/marketplace-publish@v1`. This template accepts an input argument called `resource` which needs to have a specific structure. Following is an example used to publish details of a provisioned MSSQL database.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)